misc/python/materialize/mzcompose/composition.py

```diff
@@ -330,7 +312,8 @@ def invoke(
                     raise CommandFailureCausedUIError(
                         f"running docker compose failed (exit status {e.returncode})",
                         cmd=e.cmd,
-                        stderr=e.stderr if capture_stderr == True else None,
```

Comment:
Actually this is the problem. stderr used to be just printed out if capture_stderr was not set. Now it's only captured, not printed.

Reply:
What's an appropriate fix? Shall I explicitly print the captured output myself or is there a way to specify to both print and capture the output?

Reply:
I think you have to print it too, I don't know of a good way to print and capture.

Reply:
Done, but my observation is that it won't appear exactly in the same place in the log as it would otherwise (regardless of using stdout vs. stderr and flushing).